src/sentry/static/sentry/app/components/projectSelector.jsx

```diff
@@ -268,33 +288,58 @@ class ProjectSelectorItem extends React.PureComponent {
     this.handleMultiSelect(e);
   };
 
+  clearAnimation = () => {
+    this.setState({bookmarkHasChanged: false});
```

Comment:
Hmm do we want this in setState, this would cause a re-render.

Reply:
We need to remove the animation css rule in order to re-trigger it. We trigger an update to add the "isBookmarked" class, then after the animation we update again to remove it, otherwise on the second pass the class will still be there and no animation will trigger.